disk-buffering/src/main/java/io/opentelemetry/contrib/disk/buffering/exporters/LogRecordDiskExporter.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.disk.buffering.exporters;
```

Comment:
Let's put all the public classes in `io.opentelemetry.contrib.disk.buffering`, and put the remaining stuff in `io.opentelemetry.contrib.disk.buffering.internal.*`.

In other words, I don't think there's a need for the breaking out `i.o.c.d.b.exporters` and `i.o.c.d.b.storage`.

Reply:
Sounds good to me, I've applied the changes.